.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: node_js
+node_js:
+  - 8
```

Comment:
Should we specify a more precise version of node here? We're using `"node": "^8.11.4"` in the package.json, should we use that same version here?

Reply:
Yeah, probably not a bad idea.